docs/_data/team.js

```diff
@@ -188,4 +175,17 @@ export const emeriti = [
     title: 'Senior Software Engineer',
     links: [{ icon: 'github', link: 'https://github.com/nihalgonsalves' }],
   },
+  {
+    avatar: 'https://github.com/Shinigami92.png',
+    name: 'Shinigami',
+    title: 'Senior Frontend Engineer',
+    org: 'Faker',
+    orgLink: 'https://fakerjs.dev',
+    desc: 'Passionate TypeScript enthusiast working extensively with Vue SPA.',
+    links: [
+      { icon: 'github', link: 'https://github.com/Shinigami92' },
+      { icon: 'mastodon', link: 'https://elk.zone/mas.to/@Shini92' },
+    ],
+    sponsor: 'https://github.com/sponsors/Shinigami92',
```

Comment:
Removing this line as we aren't showing sponsor links in the emeriti section
```suggestion
    sponsor: 'https://github.com/sponsors/Shinigami92',
```